config/initializers/appdev_support.rb

```diff
@@ -1,7 +1,7 @@
 AppdevSupport.config do |config|
-  config.action_dispatch = true
-  config.active_record   = true
-  config.pryrc           = :minimal
+  # config.action_dispatch = true
+  # config.active_record   = true
+  # config.pryrc           = :minimal
```

Comment:
I'm realizing now that I don't think there is an "off switch" for the `pryrc` config. `:minimal` is the default value and that's what we want for phase 1, but not phase 2. So, `appdev_support` either needs to be updated, pushed to RubyGems, and this Docker image needs to be updated and pushed to DockerHub or you remove/comment out line 7 which (I think) actually initializes the settings.

Or this whole initializer could be removed, if you're not keeping it for any particular reason.

Reply:
Seems like removing the initializer is the easiest thing to do here then.